docs/checks/matching_schema_translations.md

```diff
@@ -71,6 +71,15 @@ Add checks for eliminating translations mistakes in schema tags.
 {% endschema %}
 ```
 
+## Check Options
+
+The default configuration for this check is the following:
+
+```yaml
+MatchingSchemaTranslations:
```

Comment:
I'm a bit sad that now we got this duplicated in 3 places:

- `config/default.yml`
- the check's `initialize`
- the docs

I don't have a solution. But something to think about...

But we must update the CONTRIBUTING.md file to document this at least.

Reply:
Just an idea, if we generated the (allow some ERB in the docs) we could dynamically generate this part from the config/default.yml file.

Reply:
I feel you. I don't have a good solution to that either. If we do make a microsite I think this would be a good idea. But since now I'm linking directly to the markdown files in the docs, I don't think we can do this just yet.